### DISCUSSION TITLE
UPNP found but not an IGD

### DISCUSSION THREAD

**Discussion by SamsonSharma** [LizardByte/community#54] [Sunshine] [answered]
I'm attempting to stream across a different wifi, but this shows up 
"A UPnP device has been found,  but it wasn't recognized as an IGD"
it works on the same network as expected, I couldn't find any info on this online.

I'm on Archcraft/Arch Linux

**Comment by ReenigneArcher** [comment, marked as answer]
> it works on the same network

UPnP has no effect on LAN.

I would suggest manual port forwarding to be honest.

**Reply by SamsonSharma** [reply]
If I turn off Upnp in sunshine and my router, will it automatically look for the open ports required? or is there something else to change in the sunshine config?

**Reply by ReenigneArcher** [reply]
> If I turn off Upnp in sunshine and my router, will it automatically look for the open ports required? or is there something else to change in the sunshine config?

No, if you turn off UPnP it will not automatically configure anything. UPnP attempts to configure everything for you automatically, but it's not very reliable.
